File: patterns/javascript-patterns/pattern92.js
// Pattern 92: ...

// * * * *
//   * * *
//     * *
//       *
//     * *
//   * * *
// * * * *

let n = 7;
let x = 1;

for (let i = 1; i <= n; i++) {
    let row = '';

    for (let j = 1; j <= n / 2 + 1; j++) {
        if (j >= x) {
            row += '* ';
        } else {
            row += '  ';
        }
    }

    if (i <= n / 2) {
        x++;
    } else {
        x--;
    }

    console.log(row);
}

File: patterns/python-patterns/pattern92.py
# Pattern 92: ...

# * * * *
#   * * *
#     * *
#       *
#     * *
#   * * *
# * * * *

n = 7
x = 1

for i in range(1, n + 1):
    for j in range(1, n // 2 + 2):
        if j >= x:
            print("* ", end="")
        else:
            print("  ", end="")
    if i <= n // 2:
        x += 1
    else:
        x -= 1
    print()
